.github/workflows/integ_tests.yml

```diff
@@ -0,0 +1,38 @@
+name: Integration tests
```

Comment:
Ngl, I really liked "Build and Test" (but maybe because it's also an homage to another system) 😂 

More seriously, this workflow really does first run unit tests and then actually build the action (`npm run ci`), and only then do the integration tests 🤔 

Maybe "Build and Test" could be superseded by another name, but I'm not sure "Integration tests"/"integ_tests.yml" should be that name. Keeping "Build and Test"/"bat.yml" will also be consistent with the workflows of the same name in the other actions of this org. For this PR, let's keep the original naming and we can follow up on a potential new one for all the repos separately 😄

The steps that come after `Perform npm tasks` do serve as the integration tests, though. Perhaps they could be a separate job of their own (would need to upload/download artifacts between them), and that job could be named something like "Integration tests"? 🤔

Reply:
I agree, the name "Build and Test" should cover everything including integration tests. Apart from name related issue, I don't see an advantage of splitting up integ test into a new job.

Reply:
Sounds good to me 😄